src/Core/Managed/Shared/TelemetryClient.cs

```diff
@@ -414,7 +414,6 @@ public void Track(ITelemetry telemetry)
                 if (string.IsNullOrEmpty(telemetry.Context.InstrumentationKey))
                 {
                     TelemetryDebugWriter.WriteTelemetry(telemetry);
```

Comment:
I'm not sure this statement will be needed any longer. It may start output the same item twice in debug window.